scripts/variantstore/wdl/GvsWithdrawSamples.wdl

```diff
@@ -0,0 +1,97 @@
+version 1.0
+
+workflow GvsWithdrawSamples {
+
+  input {
+    String dataset_name
+    String project_id
+
+    Array[String] sample_names
+
+    File? withdraw_samples_gatk_override
+    String? service_account_json_path
+  }
+
+  String sample_info_table = "sample_info"
+
+  call WithdrawSamples {
+    input:
+      project_id = project_id,
+      dataset_name = dataset_name,
+      sample_info_table = sample_info_table,
+      sample_names = sample_names,
+      gatk_override = withdraw_samples_gatk_override,
+      service_account_json_path = service_account_json_path
+  }
+
+  output {
+    Int num_rows_updated = WithdrawSamples.num_rows_updated
+  }
+}
+
+task WithdrawSamples {
+  input {
+    String project_id
+    String dataset_name
+
+    String sample_info_table
+    Array[String] sample_names
+
+    # runtime
```

Comment:
Looking at the other WDLs in this repo this comment appears to be marking inputs that are used in expressions in the `runtime` section of tasks. Assuming I have that right (which I might not), `# runtime` would not apply to these two variables.

Reply:
Yeah, that's true. I'll remove the comment.